### PR TITLE
Fix/update event on undefined max participants

### LINKED ISSUE #78: Remove "event description limit"
shouldn't be a thing

### LINKED ISSUE #81: Change "Ferdig" to "Forkast" when editing an event
When editing event info, change "ferdig" to "forkast" to make it more clear what the button does.

Or maybe only say "ferdig" and lower opacity of "oppdater" and make it unclickable if the user hasn't changed anything yet.

"forkast" Should also prompt the user if they really want to forkast if they have changed things

### PR DESCRIPTION
**Pull request by FredrikMorstad**
## :hammer_and_wrench: fix removal of optional field such as maxParticipants in event update
  - fields to be removed should be set to null and not as undefined as this will be seen as an empty field by the API 
### Minor fixes:
  - fixes #81 
  - resolves #78

### CODE REVIEW

File: src/components/molecules/event/eventBody/EventBody.tsx [hunk 1]
```diff
@@ -205,9 +213,9 @@ export const EditEvent: React.FC<{ event: Event; setEdit: () => void }> = ({
                     }
                     label={'Transportation'}></ToggleButton>
                   <ToggleButton
-                    initValue={toggleActive}
-                    onChange={() => setToggleActive(!toggleActive)}
-                    label={toggleActive ? 'Active' : 'Inactive'}></ToggleButton>
+                    initValue={togglePublic}
+                    onChange={() => setTogglePublic(!togglePublic)}
+                    label={togglePublic ? 'Public' : 'Private'}></ToggleButton>
```
Comment (seblj) [inline src/components/molecules/event/eventBody/EventBody.tsx:218]: Norwegian label?

**Review by seblj — APPROVED** [review]
LGTM🥳 One small comment about changing to norwegian in label

File: src/components/molecules/event/eventBody/EventBody.tsx (hunk 1 above)
Comment (FredrikMorstad) [reply src/components/molecules/event/eventBody/EventBody.tsx:218]: yes should be Norwegian, admin pages are a little less restrictive :call_me_hand: added fix now

Comment (seblj) [reply src/components/molecules/event/eventBody/EventBody.tsx:218]: 🥳 just merge when ready then

**Pull request merged**